src/index.js

```diff
@@ -1,32 +1,136 @@
+/* eslint-disable max-len */
+
 import './style.css';
 
-const rawTasks = [{
-  index: 0,
-  completed: true,
-  description: 'wash the dishes',
-}, {
-  index: 1,
-  completed: false,
-  description: 'complete To Do list project',
-}];
+// window.localStorage.clear();
+
+let allTasks = JSON.parse(localStorage.getItem('tasks'));
+let tasks = [];
+class Task {
+  constructor(index, completed, description) {
+    this.index = index;
+    this.completed = completed;
+    this.description = description;
+  }
+}
+
+const addTask = (description) => {
+  const storedTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (storedTasks === null) {
+    allTasks = [];
+  } else {
+    allTasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+  const newTask = new Task(allTasks.length, false, description);
+  allTasks.push(newTask);
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+  window.location.reload();
+};
+
+const editTask = (index, description) => {
+  const task = allTasks.find((task) => task.index === index);
+  task.description = description;
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+};
+
+const toggleTask = (index, completed) => {
+  const task = allTasks.find((task) => task.index === index);
+  task.completed = completed;
+  localStorage.setItem('tasks', JSON.stringify(allTasks));
+};
 
-const tasks = rawTasks.sort((a, b) => a.index - b.index);
+const removeTask = () => {
+  const completedTasks = [];
+  allTasks.forEach((task) => {
+    if (task.completed === false) {
```

Comment:
- [ ] Great job so far 👏🏻 but you should create a new JavaScript file where you implement all of these new functionalities of the CRUD (create, update, delete) methods inside. Kindly make this 👨🏻‍🎨